test/deprecated/barrierCheck.good

```diff
@@ -0,0 +1 @@
+barrierCheck.chpl:5: warning: 'barrier.check()' is deprecated, please use 'barrier.pending()' instead
```

Comment:
I wonder if this should actually say: "please use '!barrier.pending()', so it's clear that the new method is the logical negation of the old.